DeepLearning/analysis.py

```diff
@@ -0,0 +1,71 @@
+import pandas as pd
+import pickle
+
+res_df = pd.read_csv("DeepLearning/results.csv", index_col="id")
+print(res_df.head())
+
+import matplotlib.pyplot as plt
+
```

Comment:
Seems like the formatter added 2 lines in a lot of places. Is that correct?
If not, please replace all 2 empty lines with a single one

Reply:
It's correct, it adds 2 lines before and after function definitions